contrib/syslog-debun

```diff
@@ -178,6 +189,15 @@ debun_finish_debug () {
 	if [ -d /proc/${pcappid} ]; then
 		kill -INT $pcappid
 	fi
+	if [ -n "$tracing" -a -z "$debug_params" ]; then
+		for i in ${tracepids[*]} ; do
+			kill -INT $i
+		done
+		sleep 2
+		for i in ${tracepids[*]} ; do
+			[ -d /proc/${i} ] && kill -9 $i 2>/dev/null
+		done
```

Comment:
As you can see, in the previous parts of this debun_finish_debug (), I try a more sophisticated way for other process killings.
The strace/truss is a new function: it's a first approach for handling the finishing.
As you can see already appeared some process related function, which is more portable.
But it's a much longer process to write a solution which works everywhere.

Now the primary goal was that, to incorporate the new enhancements as soon as possible.
Refactoring, and maybe unit tests, etc. will happen at a later time.